tests/benchmarking/test_benchmarking.py

```diff
@@ -67,6 +67,52 @@ def test_build_django_from_cache(benchmark):
         benchmark(fn)
 
 
+def test_find_descendants(large_graph, benchmark):
```

Comment:
Might be worth using benchmark.pedantic for each of these, like with the others, so we get to average out the runs. That will only apply to a local run.

Reply:
Done it.

I'm not fully sure what this does. Even without pedantic pytest-benchmark will run many iterations and take an average. What's the difference between between iterations and rounds here?